src/mlpack/tests/main_tests/dbscan_test.cpp

```diff
@@ -429,4 +429,39 @@ BOOST_AUTO_TEST_CASE(DBSCANNaiveSearchTest)
   CheckMatrices(output, naiveOutput);
 }
 
+/**
+ * Check that the assignment of cluster is different if
+ * different point selection policies.
```

Comment:
Change `different point selection policies` to `point selection policies are different`.